source/common/http/conn_manager_impl.cc

```diff
@@ -403,6 +403,10 @@ void ConnectionManagerImpl::ActiveStream::chargeStats(HeaderMap& headers) {
     return;
   }
 
+  if (CodeUtility::is1xx(response_code)) {
+    connection_manager_.stats_.named_.downstream_rq_1xx_.inc();
+    connection_manager_.listener_stats_.downstream_rq_1xx_.inc();
+  }
```

Comment:
this one still needs fixing.

Reply:
LGTM other than this one. Can we clean this up and make it consistent one way or the other?

Reply:
Oh!

I kept putting these lines together and not noticing that fix_format undid my change because I didn't change it to "else if".  "But I *swear* I fixed this!  Twice!"

Actually fixed :-)